modules/c++/math/include/math/Constants.h

```diff
@@ -23,6 +23,31 @@
 #ifndef __MATH_CONSTANTS_H__
 #define __MATH_CONSTANTS_H__
 
+#include <sys/sys_config.h>
+
+#ifdef __CODA_CPP11
+#include <math.h>
+namespace math
+{
+struct Constants
```

Comment:
Hmm... yeah I guess that's true.  So then there's no real solution aside from `#define` which defeats the purpose of trying to have static constants.  Bah.

Constants.cpp should not redefine the constants for the C++11 case though right (these are unrelated constants as far as the compiler is concerned - they're defined outside of the Constants class).